src/components/tooltip/_tooltip.scss

```diff
@@ -329,3 +329,357 @@
     }
   }
 }
+
+@mixin tooltip--experimental {
+  .#{$prefix}--tooltip--icon {
+    display: flex;
+    align-items: center;
+  }
+
+  .#{$prefix}--tooltip__label {
+    @include font-family;
+    @include typescale('epsilon');
+    display: inline-flex;
+    align-items: center;
+    color: $text-01;
+    font-weight: normal;
+    margin-bottom: $spacing-xs;
+
+    .#{$prefix}--tooltip__trigger {
+      margin-left: $spacing-xs;
+    }
+  }
+
+  .#{$prefix}--tooltip__trigger {
+    @include button-reset($width: false);
+    display: inline-flex;
+    align-items: center;
+    cursor: pointer;
+    font-size: 1rem;
+
+    &:focus {
+      @include focus-outline('border');
+      fill: $hover-primary;
+    }
+
+    path,
+    polygon,
+    circle {
+      fill: $ui-05;
+    }
+
+    &:hover,
+    &:focus {
+      color: $text-01;
+      path,
+      polygon,
+      circle {
+        fill: $ui-05;
+      }
+    }
+  }
+
+  .#{$prefix}--tooltip__label--bold {
+    font-weight: 600;
+  }
+
+  .#{$prefix}--tooltip {
+    @include layer('overlay');
+    @include reset;
+    position: absolute;
+    display: none;
+    min-width: rem(208px);
+    max-width: rem(288px);
+    // Need to add new UI color -- IBM Color Gray 80
+    background: #3d3d3d;
```

Comment:
Would it make sense to create a `$tooltip-background-color` token now, then we can update that with a UI token later?

Reply:
I'm not sure, I thought it might be easier to keep it as a hex so we can search for anything that starts with a `#` to see where we need to add variables

Reply:
Or do you mean making a vars section?

Reply:
Was thinking we could add it to the `theme_tokens` file, or wherever. Since this value is used multiple times in the file then we only have to update it once later on. 

could do
`$tooltip-background-color: #3d3d3d` if you want to keep the hex value to make it easier to find/change later. I'm fine with whatever though.

Reply:
@alisonjoseph added the variables to the main theme file